package errs

import "testing"

func Test_caller(t *testing.T) {
	caller()
}
